src/domains/product/utils/transformFilterValueToQueryString.ts

```
@@ -4,30 +4,58 @@ import {
+  };
+  const tagsQuery =
+    tags &&
+    tags
+      .map((tag) => transformTagToEng(tag))
+      .reduce(
+        (acc, tag) => ({
+          ...acc,
+          [tag]: true
+        }),
+        {}
+      );
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)